### ISSUE TITLE
Wrong link to troubleshooting documentation (port 80 being occupied) 

### ISSUE THREAD

**Issue by tmotyl** [drud/ddev#630]
## What happened (or feature request):
during `ddev start` command en error occurred with information:

```
Failed to start core-master: Unable to listen on required ports, localhost port 80 is in use,
Troubleshooting suggestions at https://ddev.readthedocs.io/en/latest/users/troubleshooting/#unable-listen 
```
Unfortunatelly the url result in 404

## What you expected to happen:

To see a troubleshooting page based on https://github.com/drud/ddev/blob/master/docs/users/troubleshooting.md

## Version: Please include the output of `ddev version`, `docker version` and the project's .ddev/config.yaml.

ddev version
web   	drud/nginx-php-fpm-local:v0.9.1
db    	drud/mariadb-local:v0.7.1      
dba   	drud/phpmyadmin:v0.2.0         
router	drud/ddev-router:v0.5.0        
commit	v0.12.0                        
domain	ddev.local                     
cli   	v0.12.0  

## Related source links or issues:
https://github.com/drud/ddev/issues/542

**Comment by rickmanelius** [comment]
Hi @tmotyl. I can confirm the issue. Readthedocs is not picking up the page. We will get this updated in the next sprint.

**Comment by tannerjfco** [comment]
This has been fixed with #641, thanks for letting us know!
